src/main/java/at/fhhagenberg/app/ECCApp.java

```diff
@@ -35,19 +39,27 @@ public class ECCApp extends Application {
     private static final int ELEVATOR_WIDTH = 280;
     private static final int FLOOR_WIDTH = 185;
 
+    private AppController mController;
+
     @Override
     public void start(Stage stage) {
         var service = createService();
+        String err = "The service could not be created!\nThe application will now shut down.";
```

Comment:
err glei in showError auslagern? in der datei werden immer nur fehler sein weshalb de applikation ned startet. da kann man eigentlich glei in showError hardcoden